### DISCUSSION TITLE
[FE] Self-destroyed request messages

### DISCUSSION THREAD

**Discussion by jpalanca** [activexray/Doplarr#42] [Ideas]
It would be amazing if there would be an option to set a timer (s/m/h) for request messages auto-deletion. It would help keep a requests channel tidy and manageable without having to rely on a secondary bot. Specially useful in tandem with Overseerr, as requested items status can be tracked directly after they’ve been picked up by it. Would it be possible to add the extra option? Cheers!

**Comment by activexray** [comment]
In the latest version, only a single small requested message is sent to the channel. The rest is ephemeral. Is that enough?
